package/etc/conf.d/conflib/syslog/app-f5_bigip.conf

```diff
@@ -27,6 +27,47 @@ block parser f5_bigip-parser() {
                 set("f5_bigip_nix", value("fields.sc4s_vendor_product"));
                 set("nix:syslog", value(".splunk.sourcetype"));
             };
+        } elif {
+            filter{
+                message('[0-9A-Fa-f]{4}\d+:\d{1,2}:\sTCL\serror:\s.+\/(?:\s[^\s]+.*ERR_NOT_SUPPORTED)?.*');
```

Comment:
regex on message is expensive, in these it looks like this only applies if program is tmm use an outer if and inside that if do if else logic for the message rules to reduce the cpu cost